pkgs/by-name/ya/yalc/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nodejs,
+  yarnBuildHook,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation rec {
```

Nitpick: `stdenvNoCC` makes it so you can build this package without pulling in a C compiler.

Another nitpick: We may want to use `finalAttrs` here.

So it would end up looking like this:
```suggestion
stdenvNoCC.mkDerivation finalAttrs: ({
```
(Make sure to change the usage of `src` in `yarnOfflineCache` to `finalAttrs.src`.

---

good call on both of those, NoCC is definitely a good closure size optimization that I missed.